apps/web/src/hooks/useStorage.ts

```diff
@@ -27,6 +27,7 @@ export const useStorage = (
   useEffect(() => {
     if (formBlob && formBlob.size > 0) {
       const url = URL.createObjectURL(formBlob);
+      // let url = 'http://localhost:3002/test.pdf';
```

Comment:
Is this an important reference for the future? If not we can remove it

Reply:
Its just for testing with a sample pdf but yes, we would remove it assuming everything else works.